api/tests/utils/fixtures_tests.py

```diff
@@ -33,3 +49,59 @@ def config_rollback():
 def app_config():
     app_config = create_app_config()
     return app_config
+
+
+@pytest.fixture
+def camera_sample():
+    camera_sample = copy.deepcopy(example)
+    return camera_sample
+
+
+@pytest.fixture
+def rollback_screenshot_camera_folder():
+    yield None
+    # Deletes the camera screenshots directory and all its content.
+    camera_screenshot_directory = os.path.join(os.environ.get("ScreenshotsDirectory"), str(example['id']))
+    if os.path.exists(camera_screenshot_directory):
+        shutil.rmtree(camera_screenshot_directory)
+
+
+@pytest.fixture
+def rollback_homography_matrix_folder():
+    yield None
+    # Deletes the homography_matrix directory and all its content.
+    raw = f'/repo/data/processor/static/data/sources/'
+    path = os.path.join(raw, str(example['id']))
```

Comment:
This will clear all the data for the camera, not just the homography matrix, right?

Reply:
Yes, `shutil.rmtree (path)` removes a directory and all its contents. In our case, inside the deleted folder, there will be the homography matrix and, sometimes, an object_log directory. But both directories need to be removed once the test is finished.